src/test/java/org/broadinstitute/hellbender/utils/SequenceDictionaryUtilsUnitTest.java

```diff
@@ -109,7 +109,7 @@
                 { Arrays.asList(CHR1_HG19, CHR2_HG19, CHR_NONSTANDARD1, CHRM_HG19, CHR_NONSTANDARD2 ), Arrays.asList(CHR1_HG19, CHR2_HG19, CHRM_HG19, CHR_NONSTANDARD2 ), DIFFERENT_INDICES, DIFFERENT_INDICES_EXCEPTION, true},
                 { Arrays.asList(CHR1_HG19, CHR_NONSTANDARD1, CHR2_HG19, CHRM_HG19, CHR_NONSTANDARD2 ), Arrays.asList(CHR1_HG19, CHR2_HG19, CHRM_HG19, CHR_NONSTANDARD2 ), DIFFERENT_INDICES, DIFFERENT_INDICES_EXCEPTION, true},
 
-                // These have isReadsToReferenceComparison == true but no overlapping intervals, so we don't expect an exception:
+                // We expect exceptions for these because the common contigs are out of order with respect to each other.
```

Comment:
Nitpick: they're not necessarily out of order with respect to each other -- they could be in the same relative order, but have different indices (eg., {"1", "2", "3"} vs {"1", "MT", "2", "3"})